src/sentry/testutils/cases.py

```diff
@@ -203,6 +203,12 @@ def make_request(self, user=None, auth=None, method=None, is_superuser=False):
         request.superuser = Superuser(request)
         if is_superuser:
             # XXX: this is gross, but its a one off and apis change only once in a great while
+            request._body = json.dumps(
+                {
+                    "superuserAccessCategory": "Edit organization settings",
+                    "superuserReason": "Edit organization settings",
+                }
+            )
             request.superuser.set_logged_in(user)
```

Comment:
This change makes the scope of the comment unclear; before it was referring to `set_logged_in`, but now it's unclear if it also refers to hacking on `request._body`. (To which the comment also would kind of apply, actually.)

```suggestion
            request._body = json.dumps(
                {
                    "superuserAccessCategory": "Edit organization settings",
                    "superuserReason": "Edit organization settings",
                }
            )

            # XXX: this is gross, but its a one off and apis change only once in a great while
            request.superuser.set_logged_in(user)
```